Commands/Base/FileManagerCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FileManager_OOP_WinForm.Commands.Base
+{
+    public abstract class FileManagerCommand
+    {
+        public abstract string Description { get; }//каждый класс должен иметь описание что именно команда делает
```

Comment:
Комментарии надо писать системным образом:
```suggestion
        /// <summary>каждый класс должен иметь описание что именно команда делает</summary>
        public abstract string Description { get; }
```
Тогда этот комментарий будет захвачен средой разработки и показан Вам при наведении мышки на свойство. Кроме того, этот комментарий будет включён в генерируемый файл комментариев.